File: app/src/main/java/volovyk/guerrillamail/data/remote/pojo/SetEmailAddressResponse.java
package volovyk.guerrillamail.data.remote.pojo;

import com.google.gson.annotations.SerializedName;

public class SetEmailAddressResponse {
    @SerializedName("sid_token")
    private String sidToken;
    @SerializedName("email_addr")
    private String emailAddress;

    public String getSidToken() {
        return sidToken;
    }

    public String getEmailAddress() {
        return emailAddress;
    }
}

File: app/src/main/java/volovyk/guerrillamail/data/remote/pojo/SetEmailAddressResponse.kt
package volovyk.guerrillamail.data.remote.pojo

import com.google.gson.annotations.SerializedName

class SetEmailAddressResponse(
    @SerializedName("email_addr")
    val emailAddress: String? = null,

    @SerializedName("sid_token")
    val sidToken: String? = null
)